Documentation/CHANGES_IN_V5.md

```diff
@@ -0,0 +1,32 @@
+# Changes in V5
+We have Made the following changes to MetroAE in v5 as compared to previous versions of MetroAE. All changes are summarised in this document.
+
+## MetroAE operation changes
+In v4 and before, MetroAE shipped with 2 modes of operation. The container and git clone. With the new release, we have made things a bit easier for the users. There is a single mode of operation. The metroae container option. This new container will be dynamically built on the users machine. The only pre-requisite for the user will be to have docker installed on their MetroAE Host Machine. The changes for moving from v4 to v5 are as below depending on type of installation. 
+
+### Moving from MetroAE git clone/download to MetroAE v5
+1. Download or git pull the latest metroae code
+2. Make sure docker is installed and running. The `docker ps` command can verify this. 
+3. Use `./metroae-container` instead of `./metroae`. All other arguments remain the same. 
```

Comment:
Should we provide an example just to be safe? Like `./metroae-container install vsds yourDeploymetName -vvv` or something?

Reply:
Sounds good. Will change